Helper/Order.php

```diff
@@ -1644,15 +1644,61 @@ protected function isCaptureHookRequest($newCapture)
 
     /**
      * @param OrderInterface $order
-     * @param                                        $captureAmount
+     * @param float          $captureAmount
      *
+     * @return void
      * @throws \Exception
      */
-    protected function validateCaptureAmount(OrderInterface $order, $captureAmount) {
+    protected function validateCaptureAmount(OrderInterface $order, $captureAmount)
+    {
         $isInvalidAmount = !isset($captureAmount) || !is_numeric($captureAmount) || $captureAmount < 0;
-        $isInvalidAmountRange = $order->getTotalInvoiced() + $captureAmount > $order->getGrandTotal();
 
-        if($isInvalidAmount || $isInvalidAmountRange) {
+        /**
```

Comment:
This is wonderful documentation, however, this is too verbose for inline documentation. Please add this to a readme or update internal documentation.

Reply:
Updated comments (moved examples to the pull request description)